client/src/components/AddAssignmentsView.js

```diff
@@ -75,17 +75,17 @@ const AddAssignmentsView = () => {
                 <AssignmentCategory 
                   key={assignment.id} 
                   assignment={assignment}  
-                  button={<Button>Edit</Button>} 
+                  button={<Button onClick={ () => navigate('/edit-assignment/' + instanceId + '/' + assignment.id) }>Edit</Button>}
```

Comment:
Could some of this be separated onto individual lines to shorten the line-length